platforms/documentation/docs/src/docs/userguide/authoring-builds/tutorial/partr1_gradle_init.adoc

```diff
@@ -87,6 +103,36 @@ When you are done with Gradle `init`, the directory should look as follows:
 <4> Build script for `app` subproject
 <5> Default Java source folder for `app` subproject
 <6> Default Java test source folder for `app` subproject
+=====
+[.multi-language-sample]
+=====
+[source, groovy]
+----
+.
+├── gradle                              // <1>
```

Comment:
I think in the latest version we generate a version catalog too. It would be good to show that in the layout. Would you do that in a separate PR?